media-gfx/icoutils/icoutils-0.31.1.recipe

```diff
@@ -0,0 +1,63 @@
+# FIXME: extresso and genresscript won't work because of missing libwww-perl
+# and that won't compile because reasons
+SUMMARY="Tools for extracting and converting Windows icon and cursor files"
+DESCRIPTION="A set of command-line programs for extracting and converting \
+images in Microsoft Windows icon and cursor files (usually with the extension \
+.ico or .cur, but they can also be embedded in executables and DLL libraries."
+HOMEPAGE="http://nongnu.org/icoutils/"
+COPYRIGHT="1998-2017 Frank Richter"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="http://savannah.nongnu.org/download/icoutils/icoutils-$portVersion.tar.bz2"
+CHECKSUM_SHA256="26e29d3c78f25d4cdf402501ac0414c51a9a092daebf6c9dee3b837dee693093"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+	icoutils = $portVersion
+	cmd:extresso = $portVersion
+	cmd:genresscript = $portVersion
+	cmd:icotool = $portVersion
+	cmd:wrestool = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
```

Comment:
I've just checked that the recipe builds fine on x86_gcc2.
I had already checked that it also builds fine on x86.

So this means we ~~can~~ should get rid of every occurence of $secondaryArchSuffix and ${secondaryArchSuffix}.

This is great news, isn't it ?

Hmm, if you could sort alphabetically the lib:lib\* lines in REQUIRES and the devel:lib\* lines in BUILD_REQUIRES that would be great!
Thanks!